CRM/Core/I18n.php

```diff
@@ -239,6 +239,27 @@ public static function languages($justEnabled = FALSE) {
     return $justEnabled ? $enabled : $all;
   }
 
+  /**
+   * Get the options available for format locale.
+   *
+   * Note the pseudoconstant can't be used as the key is the name not the value.
+   *
+   * @return array
+   */
+  public static function getLocaleOptions(): array {
+    $values = CRM_Core_OptionValue::getValues(['name' => 'languages'], $optionValues, 'label', TRUE);
+    $return = [];
+    foreach ($values as $value) {
+      $return[$value['name']] = $value['label'];
+    }
+    // Sorry not sorry.
+    // Hacking in for now since the is probably the most important use-case for
+    // money formatting in an English speaking non-US locale based on any reasonable
+    // metric.
+    $return['en_NZ'] = ts('English - New Zealand');
```

Comment:
The other ones all have the country in brackets, BUT:

On the larger r-tech, it does feel a little wrong hacking NZ in to the list. It seems like it can just be added as a real option value but without needing translations. It won't appear in the site language dropdown, because there's a built-in check that translation files exist, but it would still be available to the format_locale etc code when you retrieve the list of options values.

What I mean is just remove the hack at line 259, and then add it as a real option value (for testing purposes I just did `insert into civicrm_option_value values (null, 84, 'English (New Zealand)', 'en', 'en_NZ', null, 0,0,45, null, 0,0,1,null,null,null,null,null);`). That seems to work to get the desired functionality. Of course, for real it would go in the xml file.